google-cloud-firestore/google-cloud-firestore.gemspec

```diff
@@ -0,0 +1,36 @@
+# -*- ruby -*-
+# encoding: utf-8
+
+Gem::Specification.new do |gem|
+  gem.name          = "google-cloud-firestore"
+  gem.version       = "0.6.8"
```

Comment:
Shouldn't the version be set to `Google::Cloud::Firestore::VERSION`? (currently `0.20.0`)

Reply:
Oops. That was from the original gemspec file. It should pull the version from `Google::Cloud::Firestore::VERSION`. Fixed now.